src/app/link.service.ts

```diff
@@ -87,4 +87,24 @@ export class LinkService {
     });
   }
 
+  downloadLinks() {
+    const data = new Blob([this.linksToString()], {type: 'text/csv'});
+    const textFile = window.URL.createObjectURL(data);
+
+    chrome.downloads.download({
+      url: textFile,
+      saveAs: true
+    });
+  }
+
+  private linksToString() {
+    const links = this.linkListSource.getValue();
+    let text = 'url, visible, tag, status, content_type\n';
```

Comment:
Is it possible to iterate through all metadata fields? so we don't need to manage a separate list here?

Reply:
I don't think we want to save all of the metadata fields though. For example highlightOn and full Source don't seem like things that we want to save.